gettingStarted/index.md

```diff
@@ -35,6 +35,27 @@ Go to Servers => Nodes and add a new node. After adding a node, go to the setup
 
 After the installation is finished, you should see information about the node in the information page of the node. If it shows data, the node has been successfully installed.
 
+> Please note that your have an File in ``/etc/moonlight/config.json`` you must Delete the File to Run the Deamon Command.
+
+**Deamon Error Error**:
+
+If you Deamon can't install and you get this error:
+```
+E: Unable to locate package dotnet-sdk-7.0
+E: Couldn't find any package by glob 'dotnet-sdk-7.0'
+```
+Then you need to install the Dotnet SDK 7.0 
+> ### Link to microsoft: 
+> - https://learn.microsoft.com/de-de/dotnet/core/install/linux-ubuntu-install?pivots=os-linux-ubuntu-2204&tabs=dotnet7
+> - https://learn.microsoft.com/de-de/dotnet/core/install/linux-ubuntu-install?pivots=os-linux-ubuntu-2204&tabs=dotnet7#ubuntu-2204
+
+#### Command - Copy Paste:<br>
+*(Work on Ubuntu 22.4)*
```

Comment:
This step is not needed on Ubuntu 22.04. Works fine for me withou adding the repos

Reply:
> Yes but on some server not... (My Hetzner Ubuntu 22.4 server i must install dotnet 7.0) And some other user have this Error...

Reply:
> > Yes but on some server not... (My Hetzner Ubuntu 22.4 server i must install dotnet 7.0) And some other user have this Error...

i also use a Hetzner Server and it works fine. With every install now. MB do not blind use the default Ubuntu repos and the the correct one ;)

Reply:
Yes but if you Install Ubuntu 22.4 from the console i have this Problem and if you take a look at the discord other users have the same! 
Masu can merge it but doesn't have to. I think it makes sense but that's her decision.

Reply:
> Yes but if you Install Ubuntu 22.4 from the console i have this Problem and if you take a look at the discord other users have the same! Masu can merge it but doesn't have to. I think it makes sense but that's her decision.

Thats not a ML problem; edit the source.list after install. The default source.list is crap